src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -117,7 +122,8 @@
         "Project ID is required to access the auth service. Use a service account credential or "
             + "set the project ID explicitly via FirebaseOptions. Alternatively you can also "
             + "set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.");
-    this.baseUrl = String.format(ID_TOOLKIT_URL, projectId);
+    this.userBaseUrl = String.format(ID_TOOLKIT_URL, "v1", projectId);
+    this.tenantBaseUrl = String.format(ID_TOOLKIT_URL, "v2", projectId);
```

Comment:
I noticed that Go SDK uses v2beta1. But if this is indeed supported, then it's ok (and even preferred).

Reply:
Yeah, I did notice that the Go implementation was using the beta. But I think the non-beta version is supported: https://identitytoolkit.googleapis.com/$discovery/rest?version=v2.